README.md

```diff
@@ -144,7 +144,7 @@ Help improve Public Lab software!
 
 ## First Time?
 
-New to open source/free software? Here is a selection of issues we've made **especially for first-timers**. We're here to help, so just ask if one looks interesting : https://publiclab.github.io/community-toolbox/#r=all
+New to open source/free software? Here is a selection of issues we've made **especially for first-timers**. We're here to help, so just ask if one looks interesting : https://code.publiclab.org
 
 
 We also have a slightly larger list of easy-ish but small and self-contained issues: https://github.com/publiclab/plots2/labels/help-wanted
```

Comment:
Hi @igniteeng000, you are supposed to remove this line too, as per FTO. Also, the FTO doesn't mention any changes except readme so could you please exclude those changes. Thanks!